R/pkg/R/WindowSpec.R

```diff
@@ -127,6 +127,19 @@ setMethod("orderBy",
 #' "0" means "current row", while "-1" means the row before the current row, and "5" means the
 #' fifth row after the current row.
 #'
+#' We recommend users use \code{Window.unboundedPreceding}, \code{Window.unboundedFollowing},
+#' and \code{Window.currentRow} to specify special boundary values, rather than using long values
+#' directly.
+#'
+#' A range-based boundary is based on the actual value of the ORDER BY
+#' expression(s). An offset is used to alter the value of the ORDER BY expression, for
```

Comment:
I'm having trouble with this text -- not sure it 'alters' the values of an expression? ", for" should begin a new sentence. "order by" isn't capitalized consistently. Is this taken from a Scala example?